frontend/src/Pages/Signup.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import axios, { AxiosResponse } from "axios";
+import Fieldset from "../components/Form/Fieldset";
+import FormInput from "../components/Form/FormInput";
+import Button from "../components/Button";
+
+export default function Signup() {
+  const [username, setUsername] = useState<string>("");
+  const [password, setPassword] = useState<string>("");
+
+  const signup = () => {
+    axios
+      .post(
+        "http://localhost:5050/signup",
+        {
+          username,
+          password,
+        },
+        {
+          withCredentials: true,
+        }
+      )
+      .then((res: AxiosResponse) => {
+        if (res.data === "success") {
+          window.location.href = "/login";
+        }
+      });
+  };
+
+  return (
+    <>
+      <h1>Register</h1>
+      <Fieldset>
```

Comment:
As this is a form, I actually expect a `<form>` tag, which has the `onSubmit` event listener which in return calls the `signup` function

Reply:
As you can see the LOGIN and the SIGNUP page are very simular.
Keep the DRY principle in mind. How could you refactor these two files into one and only have a differentiation between loggin in or signin up